### PR TITLE
portmod: 2.0.3 -> 2.1.0

### PR DESCRIPTION
**Pull request by marius851000**
###### Motivation for this change
A new version was released

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [ ] [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - [ ] and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - [ ] or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Tried to update the graphics-overhaul mod set. As a result, added openmw to the binary dependancies for ``bsatool``.
This mod set contain other bugs that I'll upstream later. Portmod comes with it's own updating method for mod repository.

*(Ooh... didn't knew this package tests stuff...)*

### CODE REVIEW

**Comment by Artturin** [comment]
@ofborg eval

**Comment by marius851000** [comment]
Why does the bot complain about a "picoscope" program that isn't even in my branch ? Anyway, that doesn't seems to be related to my changes.

**Comment by Artturin** [comment]
the eval was broken for a bit so all pr's made during that time got the eval error

i've restarted the basic eval checks

File: pkgs/games/portmod/default.nix [hunk 1]
```diff
@@ -1,22 +1,22 @@
 { lib, callPackage, python3Packages, fetchFromGitLab, cacert,
   rustPlatform, bubblewrap, git, perlPackages, imagemagick, fetchurl, fetchzip,
-  jre, makeWrapper, tr-patcher, tes3cmd, fetchpatch }:
+  jre, makeWrapper, tr-patcher, tes3cmd, fetchpatch, openmw }:
```
Comment (SuperSandro2000) [inline pkgs/games/portmod/default.nix:3]:
```suggestion
, jre, makeWrapper, tr-patcher, tes3cmd, openmw }:
```

File: pkgs/games/portmod/default.nix [hunk 2]
```diff
@@ -1,22 +1,22 @@
 { lib, callPackage, python3Packages, fetchFromGitLab, cacert,
   rustPlatform, bubblewrap, git, perlPackages, imagemagick, fetchurl, fetchzip,
```
Comment (SuperSandro2000) [inline pkgs/games/portmod/default.nix:2]:
```suggestion
{ lib, callPackage, python3Packages, fetchFromGitLab, cacert
, rustPlatform, bubblewrap, git, perlPackages, imagemagick, fetchurl, fetchzip
```

**Pull request merged**